vendor-extra/ViewsBundle/src/Views/SimplifiedInlineVisitor.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Libero\ViewsBundle\Views;
+
+use FluentDOM\DOM\Element;
+use function is_string;
+use function sprintf;
+
+trait SimplifiedInlineVisitor
```

Comment:
Need a better name, would also apply to thinks like paragraphs (another thing where you don't know the template up front) which aren't inline.